content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,369 @@
+---
+title: API Priority and Fairness
+content_template: templates/concept
+min-kubernetes-server-version: v1.18
+---
+
+{{% capture overview %}}
+
+Controlling the behavior of the Kubernetes API server in an overload situation
+is a key task for cluster administrators. The {{< glossary_tooltip
+term_id="kube-apiserver" text="kube-apiserver" >}} has some controls available
+(i.e. the `--max-requests-inflight` and `--max-mutating-requests-inflight`
+command-line flags) to limit the amount of outstanding work that will be
+accepted, preventing a flood of inbound requests from overloading and
+potentially crashing the API server, but these flags are not enough to ensure
+that the most important requests get through in a period of high traffic.
+
+The API Priority and Fairness feature is an alternative to the
+aforementioned max-inflight limitations.  This new feature classifies
```

Comment:
I put the explicit "new" qualifier in there because the immediately preceding noun can also be considered a "feature" but is not the feature I mean.

Reply:
Agree with both of you.
* I'd call P&F an improvement, not just an alternative feature in the preceding sentence.
* Rather than "new feature", which doesn't sound right for docs that will live many releases in the future, maybe define "P&F" as an abbreviation in the prior sentence and use it here?

Reply:
The full name is "API Priority and Fairness", and I think the "API" qualifier is meaningful because we have considered other places to put priority and fairness (e.g., in work queues), so I am inclined to try to not drop the "API" part.  IMHO, the most natural abbreviation is "APF".